### PR TITLE
Improve open-source contribution infrastructure

### LINKED ISSUE #71: [epic] Improve open-source contribution infrastructure
- [x] #79
- [x] #121
- [x] #123
- [x] Improve documentation by:
    - [x] Add installation pages (fixed with #120)
    - [x] Add automatic API docs (fixed with #120)
    - [x] Set up infrastructure for tutorials/worked examples (fixed with #122)

### PR DESCRIPTION
**Pull request by kallewesterling**
Fixes #71

### CODE REVIEW

**Comment by kallewesterling** [comment]
@mastoffel This is just to resolve the last few comments from #71 where we had discussed plugging this information in from the related issues in comments. I have also connected up the code of conduct and the contributions guide to the documentation.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,7 +30,9 @@ The easiest way to get involved with the active development of AutoEmulate is to
 
 ## How to Contribute
 
-We welcome contributions of all kinds, be it code, documentation, or community engagement. We encourage you to read through the following sections to learn more about how you can contribute to
+We welcome contributions of all kinds, be it code, documentation, or community engagement. We encourage you to read through the following sections to learn more about how you can contribute to the package.
+
+We are always interested in adding more sampling/experimental design strategies (see https://github.com/alan-turing-institute/autoemulate/issues/6) and more simulation data packages (see https://github.com/alan-turing-institute/autoemulate/issues/4).
```
Comment (mastoffel) [inline CONTRIBUTING.md:35]: I think we can take out the first bit about design strategies. Most sampling/experimental design strategies are implemented already in some software, so we just need to chose a few more.

But it'd be great to have more simulations, or simulation input/output datasets from any field, let's maybe focus this on the second part.

**Review by mastoffel — COMMENTED** [review]
Great work! Just a minor comment, then happy to merge!

File: CONTRIBUTING.md (hunk 1 above)
Comment (kallewesterling) [reply CONTRIBUTING.md]: See https://github.com/alan-turing-institute/autoemulate/pull/156/commits/ebaa98a93fb6315fe8a87176de024c9b764bd60b

**Pull request merged**